public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -12,6 +14,23 @@ import { onRemovePanel, toggleVizPanelLegend } from './PanelMenuBehavior';
 
 export function setupKeyboardShortcuts(scene: DashboardScene) {
   const keybindings = new KeybindingSet();
+  let vizPanelKey: string | null = null;
+
+  const panelAttentionSubscription = appEvents.getStream(SetPanelAttentionEvent).subscribe((event) => {
```

Comment:
there is a simpler version where you can just call subscribe directly on appEvents:
`appEvent.subscribe(SetPanelAttentionEvent,  (event) => { ...`